test/integration/controllers/user.spec.js

```diff
@@ -4,16 +4,20 @@ const { USER_NOT_FOUND } = require('~/consts/errors')
 const { expectError } = require('~/test/helpers')
 
 const endpointUrl = '/users/'
-const nonExistingUserId = '6329a8c501bd35b52a5ecf8c'
 
 let testUser = {
   role: ['student'],
   firstName: 'john',
   lastName: 'doe',
   email: 'johndoe@gmail.com',
-  password: 'supersecretpass'
+  password: 'supersecretpass',
+  isEmailConfirmed: true,
```

Comment:
why those fields are needed to create a user?

Reply:
isEmailConfirmed
isFirstLogin
lastLogin

Reply:
because i will filter by those fields

Reply:
how are they connected to a user creation?

Reply:
removed